hack/components/bump-kubemacpool.sh

```diff
@@ -85,6 +85,12 @@ EOF
 
     (
         cd config/cnao
+
+        echo setting pods to opt-in mode
+        cp ../default/mutatepods_opt_in_patch.yaml ../release/mutatepods_opt_mode_patch.yaml
+        echo setting vms to opt-in mode
+        cp ../default/mutatevirtualmachines_opt_in_patch.yaml ../release/mutatevirtualmachines_opt_mode_patch.yaml
+
         cp ../release/mutatepods_opt_mode_patch.yaml .
         cp ../release/mutatevirtualmachines_opt_mode_patch.yaml .
```

Comment:
We can just do 
```bash
        echo setting pods to opt-in mode
        cp ../default/mutatepods_opt_in_patch.yaml mutatepods_opt_mode_patch.yaml
        echo setting vms to opt-in mode
        cp ../default/mutatevirtualmachines_opt_in_patch.yaml mutatevirtualmachines_opt_mode_patch.yaml
```

Reply:
yeah sure..

the reason it's like that is that I wanted to keep the same as the kubemacpool README instructions.. 
but you're right it's just redundant